gradle-plugin/openshift/pom.xml

```diff
@@ -79,6 +79,10 @@
             <groupId>org.gradle</groupId>
             <artifactId>gradle-model-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.gradle</groupId>
+            <artifactId>gradle-platform-jvm</artifactId>
+        </dependency>
```

Comment:
This is used for testing, could you move it to the right section
It's also good if you could preserve the same order as we have in the gradle-plugin/kubernetes/pom.xml file (so basically just after the new gradle-base-services... entry)